datasets/_global/global_energy_own/crawler.py

```diff
@@ -139,8 +132,7 @@ def crawl_rel(context: Context, row: Dict[str, str], skipped: Set[str]):
     context.audit_data(
         row, ignore=["subject_entity_name", "interested_party_name", "index"]
     )
-    if subject_entity_id not in skipped and interested_party_id not in skipped:
-        context.emit(ownership)
+    context.emit(ownership)
```

Comment:
the only thing her enow is that ```
2024-10-01 10:42:31 [info     ] energy-monitor-e100000002239 references itself via ownershipOwner -> energy-monitor-f8aae69cdc592f294857e55cb05835df54719b87 -> asset [global_energy_own] dataset=global_energy_own
2024-10-01 10:42:31 [info     ] energy-monitor-e100000002239 references itself via ownershipAsset -> energy-monitor-f8aae69cdc592f294857e55cb05835df54719b87 -> owner [global_energy_own] dataset=global_energy_own
```

Reply:
Cool - if you look in the data, does it look like they've made some kind of mistake listing an entity as an owner of itself?

Reply:
yup. it's happening for` DTE Electric Co`

Reply:
cool - let's add DTE to a set of self-owns to ignore and skip when asset and ownder ID equal and are in the known self-owns

Reply:
perhaps also email them and ask if that's intended. Then they might improve their data